zwave_js_server/model/node.py

```diff
@@ -454,12 +485,26 @@ def handle_value_removed(self, event: Event) -> None:
     def handle_value_notification(self, event: Event) -> None:
         """Process a node value notification event."""
         # append metadata if value metadata is available
-        value = self.values.get(_get_value_id_from_dict(self, event.data["args"]))
+        value = None
+        value_notification = ValueNotification(self, event.data["args"])
+
+        # If endpoint isn't provided, we need to search for the Value
+        if value_notification.endpoint is None:
```

Comment:
If endpoint is `None` we know that the endpoint part in the value_id will be `"00"`. Why can't we look up the value with that info?

Reply:
Is it that some events have the endpoint and some don't over time?

Reply:
Maybe clarify with Al? The [docs](https://zwave-js.github.io/node-zwave-js/#/api/valueid?id=valueid) say that a missing endpoint means it's the root endpoint.

> endpoint - (optional) The index of the node's endpoint (sub-device). 0, undefined and **omitting the index** addresses the root device (in fact, these options can be used interchangeably). An index >= 1 addresses one of the single endpoints.

That's at least for setting values, hopefully it's consistent with receiving values?

Reply:
the event links to that same part of the docs, so maybe we can simplify this and say that if endpoint isn't included, we use endpoint 0. @MartinHjelmare maybe I am misunderstanding what your original question was but every value should have an endpoint so 00 should never resolve. Perhaps we should default to `0` instead of `00` in the value ID generation. That could be a breaking change tohugh.

Reply:
What i saw is that the metadata updates from the interview have endpoint id 0 for the central scene value metadata, while the value notification has no endpoint, so since the default was 00, 00 != 0.

Reply:
unsure if the same is true for GE switches with the basic notifications... but maybe those do include an endpoint in the notification